src/communication/xcConnection.ts

```diff
@@ -34,7 +29,9 @@ export class DefaultConnection implements Connection {
             log.debug("getModel request failed");
             log.debug(err);
         };
-        session.init(openListener, errorListener);
+        let closeListener = (_: CloseEvent) => {
```

Comment:
Because these methods are only used in xcspy web. The user controls these calls by clicking on a button. I think it is more confortable in the xcspy web to let the user decide when to remake a call.